### DISCUSSION TITLE
Add latest Thycotic.SecretServer (v0.60.6) zipped module to the "Releases" page.

### DISCUSSION THREAD

**Discussion by MaximoTrinidad** [thycotic-ps/thycotic.secretserver#270] [General]
Following the issue(https://github.com/thycotic-ps/thycotic.secretserver/releases/tag/v0.60.6) I posted. I'm a little disappointed because it was closed without any resolution or confirmation that it works.

So someone provided a link to get the zipped module but due to my organization hardening desktop, I can't get to it. 
My Organization allows me to access GitHub Repo, as well as the PowerShell Gallery.

My Suggestion! PLEASE include the latest Thycotic.SecretServer (v0.60.6) zipped module to the "Releases" page:
https://github.com/thycotic-ps/thycotic.secretserver/releases/tag/v0.60.6

This will make it much easier for everyone to get to it!

**Comment by EndlessTrax** [comment]
It is already there to download. Check the assets section.

**Reply by MaximoTrinidad** [reply]
I meanty the whole module folder, not the source code. As I try to explain before ... if you guys having issues with PowerShell Gallery! PLEASE, make the module already built in the asset section.  And again, how hard can this be? 

Just give us (Customers) a quick way to get it.

Someone (@jagger) already provided the module zipped: https://thyproservices.z20.web.core.windows.net/Thycotic.SecretServer.zip 
Unfortunately, my organization is blocking the site. making all my progress to a stop. 

Please, just add this zipped file to assets section. 

This way there's no need for PowerShell Gallery.  You will make many customers happy!!
:)

**Reply by EndlessTrax** [reply]
It is the module, not the source code.

**Reply by EndlessTrax** [reply]
![Screenshot 2022-04-18 095537](https://user-images.githubusercontent.com/17141891/163818527-1a64c2ca-127e-4f4c-9ae7-abb0eec6cc99.png)

**Reply by MaximoTrinidad** [reply]
And again... *.rar files are blocked, and not supported for download on our business desktop machines.
I can download *.zip from GitHub. 

Can you guys follow the Github "Assets section" pattern? Here's an example from the PowerShell repo:

![pwshrepo_2022-04-18_10-32-25](https://user-images.githubusercontent.com/19892435/163823821-3b33f0d1-58e3-4ae5-b171-8a15bb646de9.png)

**Reply by EndlessTrax** [reply]
Added a `.zip` version too. 

I'm curious, what's your company's rationale for blocking `.rar` files but not `.zip`?

**Reply by MaximoTrinidad** [reply]
@EndlessTrax   
Well, our financial institution also blocks .zip files for the purpose of eliminating threads. I had to request special permission to access the file. Our Security department allows us to access the PowerShell Gallery repo.  Anything in GitHhub we need to request permission.

I hope you guys fix the issues with GitHub! 
:)

**Comment by MaximoTrinidad** [comment]
Thank YOU!!

This is AWESOME!!

![thycoticModule zipped_2022-04-18_10-32-25](https://user-images.githubusercontent.com/19892435/163914356-c7b35788-fbb5-4ed1-8fb2-68d419d44384.png)
